maketgz

```diff
@@ -186,7 +186,8 @@ retar() {
   cd "$tempdir"
   gzip -dc "../$targz" | tar -xf -
   find curl-* -type f -exec touch -c -t "$filestamp" '{}' +
-  find curl-* -type f | sort | tar --create --format=ustar --owner=0 --group=0 --numeric-owner --files-from - | gzip --best --no-name > out.tar.gz
+  find curl-* -depth -type d -exec touch -c -t "$filestamp" '{}' +
```

Comment:
Instead of two find invokes, one for the files and one for the dirs, can't it just be a single one without `-type` ?

Reply:
Good point!